{
  "artifacts": [],
  "command": "main",
  "experiment": {
    "base_dir": "/home/lgrootde/Generative-re-tests",
    "dependencies": [
      "datasets==2.17.1",
      "evaluate==0.4.1",
      "numpy==1.26.4",
      "sacred==0.8.5",
      "torch==2.2.1",
      "transformers==4.38.1",
      "wasabi==1.1.2"
    ],
    "mainfile": "run.py",
    "name": "run",
    "repositories": [
      {
        "commit": "6f0174a50fcecce566d1b08abd5412341713c430",
        "dirty": true,
        "url": "https://github.com/ontox-hu/Generative-re-tests"
      }
    ],
    "sources": [
      [
        "run.py",
        "_sources/run_1e3fabb1c6b0e62b841ed1ddcd024e0b.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/sacred/config/captured_function.py\", line 42, in captured_function\n    result = wrapped(*args, **kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/run.py\", line 260, in main\n    trainer.train()\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/transformers/trainer.py\", line 1624, in train\n    return inner_training_loop(\n           ^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/transformers/trainer.py\", line 1961, in _inner_training_loop\n    tr_loss_step = self.training_step(model, inputs)\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/transformers/trainer.py\", line 2902, in training_step\n    loss = self.compute_loss(model, inputs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/transformers/trainer.py\", line 2925, in compute_loss\n    outputs = model(**inputs)\n              ^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/torch/nn/modules/module.py\", line 1511, in _wrapped_call_impl\n    return self._call_impl(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/torch/nn/modules/module.py\", line 1520, in _call_impl\n    return forward_call(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/transformers/models/t5/modeling_t5.py\", line 1711, in forward\n    encoder_outputs = self.encoder(\n                      ^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/torch/nn/modules/module.py\", line 1511, in _wrapped_call_impl\n    return self._call_impl(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/torch/nn/modules/module.py\", line 1520, in _call_impl\n    return forward_call(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/transformers/models/t5/modeling_t5.py\", line 1115, in forward\n    layer_outputs = layer_module(\n                    ^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/torch/nn/modules/module.py\", line 1511, in _wrapped_call_impl\n    return self._call_impl(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/torch/nn/modules/module.py\", line 1520, in _call_impl\n    return forward_call(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/transformers/models/t5/modeling_t5.py\", line 695, in forward\n    self_attention_outputs = self.layer[0](\n                             ^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/torch/nn/modules/module.py\", line 1511, in _wrapped_call_impl\n    return self._call_impl(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/torch/nn/modules/module.py\", line 1520, in _call_impl\n    return forward_call(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/transformers/models/t5/modeling_t5.py\", line 602, in forward\n    attention_output = self.SelfAttention(\n                       ^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/torch/nn/modules/module.py\", line 1511, in _wrapped_call_impl\n    return self._call_impl(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/torch/nn/modules/module.py\", line 1520, in _call_impl\n    return forward_call(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/transformers/models/t5/modeling_t5.py\", line 565, in forward\n    attn_weights = nn.functional.dropout(\n                   ^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/torch/nn/functional.py\", line 1268, in dropout\n    return _VF.dropout_(input, p, training) if inplace else _VF.dropout(input, p, training)\n                                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 128.00 MiB. GPU 0 has a total capacity of 21.99 GiB of which 117.88 MiB is free. Including non-PyTorch memory, this process has 21.86 GiB memory in use. Of the allocated memory 21.57 GiB is allocated by PyTorch, and 25.09 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)\n"
  ],
  "heartbeat": "2024-03-04T21:10:45.880027",
  "host": {
    "ENV": {},
    "cpu": "Intel(R) Xeon(R) Platinum 8358 CPU @ 2.60GHz",
    "gpus": {
      "driver_version": "545.23.08",
      "gpus": [
        {
          "model": "NVIDIA A10",
          "persistence_mode": false,
          "total_memory": 23028
        }
      ]
    },
    "hostname": "jupyternoteboo.src-oraclefreeb.src.surf-hosted.nl",
    "os": [
      "Linux",
      "Linux-5.15.0-1052-oracle-x86_64-with-glibc2.31"
    ],
    "python_version": "3.12.1"
  },
  "meta": {
    "command": "main",
    "config_updates": {},
    "named_configs": [],
    "options": {
      "--beat-interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--id": null,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print-config": false,
      "--priority": null,
      "--queue": false,
      "--s3": null,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [],
      "help": false,
      "with": false
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2024-03-04T21:10:36.050651",
  "status": "FAILED",
  "stop_time": "2024-03-04T21:10:45.880767"
}